iree/hal/host/host_submission_queue.h

```diff
@@ -125,20 +77,21 @@ class HostSubmissionQueue {
   };
   struct Submission : public IntrusiveLinkBase<void> {
     absl::InlinedVector<PendingBatch, 4> pending_batches;
-    FenceValue fence;
   };
 
   // Returns true if all wait semaphores in the |batch| are signaled.
-  bool IsBatchReady(const PendingBatch& batch) const;
+  // Returns any failure that status set on any of the wait semaphores.
```

Comment:
Having trouble parsing this. How about rewording?

`Returns if the batch is ready, or returns any failure status set on any of the wait semaphores."